stl/inc/numeric

```diff
@@ -14,6 +14,11 @@
 #include <limits>
 #endif // _HAS_CXX17
 
+#ifdef __cpp_lib_concepts
+#include <algorithm>

```

Comment:
Including `<algorithm>` might be a bit heavy. We only need the definition of `out_value_result`, so maybe that could be extracted into a separate file?

Reply:
I think `out_value_result` should be moved to `<xutility>`, which is included by both `<algorithm>` and `<numeric>` in MSVC STL. Note that `in_out_result` is already there.